Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -107,19 +89,19 @@
                                     replaceRoleQuery(parameterMap.$filter, getRoleKey(role), role.id);
                             });
 
-                            parameterMap.$filter = replaceOptionQuery(parameterMap.$filter,
+                            parameterMap.$filter = Helpers.OdataQueryHelper.replaceOptionQuery(parameterMap.$filter,
```

Comment:
I moved this method to a helper, because I wanted to reuse it in the ItContractOverview, but I dropped the idea later as it wouldn't work.

Should I revert this change?